weaviate/users/users.go

```diff
@@ -1,6 +1,16 @@
 package users
 
-import "github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+import (
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+	"github.com/weaviate/weaviate/entities/models"
+)
+
+type UserType string
+
+const (
+	UserTypeDb   UserType = UserType(models.UserTypeDb)
```

Comment:
```suggestion
	UserTypeDB   UserType = UserType(models.UserTypeDb)
```
Same as the note on initialisms above ☝️ 

I see that `models` doesn't adhere to that, but we can maintain that style in the client regardless, just like we do below with `UserTypeOIDC`